Backend/WebApi.Weather/Minutely15OptionsParameter.cs

```diff
@@ -0,0 +1,7 @@
+namespace WebApi.Weather;
+
+public enum Minutely15OptionsParameter
+{
+    weathercode,
+    temperature_2m
```

Comment:
https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations

Reply:
wykorzytuje to jako lista parametrów jakie dokładnie przyjmuje Api, nie mogę zmienić nazw co sugerujesz?